logic/raid_list.php

```diff
@@ -35,9 +39,12 @@ function raid_list($update)
                     LEFT JOIN   gyms
                     ON          raids.gym_id = gyms.id
                     LEFT JOIN   users
-                    ON          raids.user_id = users.user_id
-		      WHERE     raids.id = {$iqq}
-                      AND       end_time>UTC_TIMESTAMP()
+		    ON          raids.user_id = users.user_id
+		     WHERE     ( raids.id = {$iqq} or gyms.gym_name like '%{$db->real_escape_string($update['inline_query']['query'])}%' )
+		      AND       end_time>UTC_TIMESTAMP()
+                    ORDER BY
+		    ( 6371000 * acos( cos( radians( " . $config->INLINESEARCH_LATITUDE . " ) ) * cos( radians( gyms.lat ) ) * cos( radians( gyms.lon ) - radians(".  $config->INLINESEARCH_LONGITUDE . ") ) + sin( radians(" . $config->INLINESEARCH_LATITUDE . ")) * sin(radians(gyms.lat))))
+		    LIMIT 8
```

Comment:
Is this a hard limitation from somewhere or should it be configurable? If it's a hard limit that should be documented here with a comment.